src/scripts/main.js

```diff
@@ -1,8 +1,27 @@
 'use strict';
 
 const people = require('./lib/people');
+const dashboard = document.querySelector('.dashboard');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const fullPeopleData = people.map(person => {
+  person.age = person.died - person.born;
+  person.century = Math.ceil(person.died / 100);
+  person.sex = person.sex === 'm' ? 'Male' : 'Female';
```

Comment:
You could simply write all those expressions in your `insertAdjacentHTML` method